lib/components/dropdown.vue

```diff
@@ -28,7 +28,7 @@
             <span class="sr-only">{{toggleText}}</span>
         </b-button>
 
-        <div :class="['dropdown-menu',{'dropdown-menu-right': right}]"
+        <div :class="['dropdown-menu',{'dropdown-menu-right': right, show: visible}]"
```

Comment:
Since it now appears on both the `.dropdown` container element, and the `.dropdown-menu` container